rpm/SPECS/openresty.spec

```diff
@@ -108,11 +110,8 @@ services, and dynamic web gateways.
 
 %build
 ./configure \
-    --with-cc-opt="-I%{orprefix}/openssl/include" \
-    --with-ld-opt="-L%{orprefix}/openssl/lib -Wl,-rpath,%{orprefix}/openssl/lib" \
-    --with-zlib=../zlib-%{zlib_version} \
-    --with-pcre=../pcre-%{pcre_version} \
-    --with-pcre-opt="-DSUPPORT_UTF" \
+    --with-cc-opt="-I%{orprefix}/include -I%{orprefix}/openssl/include" \
+    --with-ld-opt="-L%{orprefix}/%{_lib} -Wl,-rpath,%{orprefix}/%{_lib}" \
```

Comment:
These paths should be updated for the right prefixes of zlib, pcre, and openssl (they should have their own sub-directories under `%{orprefix}/` instead of flattening directly.